lib/avpair.c

```diff
@@ -843,14 +843,17 @@ int rc_avpair_tostr (rc_handle const *rh, VALUE_PAIR *pair, char *name, int ln,
 	switch (pair->type)
 	{
 	    case PW_TYPE_STRING:
-	    	lv--;
-	    	pos = 0;
+		lv--;
+		pos = 0;
+		unsigned int slen;
```

Comment:
I prefer if declarations are done on top of the blocks. Would you like to move it on top?